servicenow/stackstate_checks/servicenow/servicenow.py

```diff
@@ -95,14 +138,20 @@ def _process_components(self, instance_config, timeout):
         state = self._collect_components(instance_config, timeout)
 
         for component in state['result']:
-            comp_name = component['name'].encode('utf-8')
+            component = self.filter_empty_metadata(component)
+            identifiers = []
+            comp_name = component['name']
             comp_type = component['sys_class_name']
-            external_id = "urn:servicenow:{}:{}".format(comp_type, component['sys_id'])
-            data = {
-                "sys_id": component['sys_id'],
```

Comment:
is the `sys_id` now always in the returned data?

Reply:
yes it's a mandatory attribute which will be always present for every component.